psi-sample/common/src/main/java/com/r3/conclave/sample/common/InputDataSerializer.java

```diff
@@ -0,0 +1,61 @@
+package com.r3.conclave.sample.common;
```

Comment:
How did you get kryo to work without having to specify the serialization/reflection config?

Reply:
I have written custom serializers and de serializers which is used by Kryo to serialize and deserialize the data from client to the enclave and vice versa. Since Kryo uses this class for serialization/de serialization, reflection is also not required internally and hence enclave does not complian.

Reply:
cool! nice workaround